src/services/auth.service.ts

```diff
@@ -19,7 +19,7 @@ export class AuthorizationService {
     return this.instance;
   }
 
-  async getUserFromToken( token: string ): Promise<ITokenPayload | undefined> {
+  async getUserFromToken(token: string): Promise<ITokenPayload | undefined> {
     try {
       if (!token) {
```

Comment:
Can we check if we can stop the request if the token is not present at the controller level?